tests/features/java/openjdk.feature

```diff
@@ -79,6 +89,12 @@ Feature: Miscellaneous OpenJDK-related unit tests
 
   @ubi8/openjdk-8
   @ubi8/openjdk-8-runtime
+    Scenario: Ensure LANG is defined and contains UTF-8
```

Comment:
I don't think we should split these tests around jdk8. We don't need to if we don't change JAVA_HOME.